src/plugins/plugin.legend.js

```diff
@@ -106,6 +107,9 @@ var Legend = Element.extend({
 		// Contains hit boxes for each dataset (in dataset order)
 		this.legendHitBoxes = [];
 
+		// Contains the currently hovered legend item
+		this.hoveredItem = null;
```

Comment:
We try to make most new fields private by default unless it's absolutely necessary to expose them to the user. Would you be able to do that here? I.e. change it to:

```
/**
 * @private
 */
this._hoveredItem = null;
```